test/cmdlineTests/standard_import_ast_missing_storage_layout_specifier_expression/output.json

```diff
@@ -0,0 +1,20 @@
+{
+    "errors": [
+        {
+            "component": "general",
+            "formattedMessage": "Unknown exception during compilation: /solidity/libsolidity/interface/StandardCompiler.cpp(1383): Throw in function solidity::Json solidity::frontend::StandardCompiler::compileSolidity(InputsAndSettings)
+Dynamic exception type: boost::wrapexcept<solidity::util::Exception>
+std::exception::what: Failed to import AST: Expected field 'baseSlotExpression' is missing.
+[solidity::util::tag_comment*] = Failed to import AST: Expected field 'baseSlotExpression' is missing.
+",
```

Comment:
Ouch, so we're not even handling those AST asserts in Standard JSON mode? Let's replace it with a test that uses the CLI interface, the ICE output is not meant to be tested.

Also please mention in #15854 that currently only the `CommandLineInterface` catches the exceptions in any way: https://github.com/ethereum/solidity/blob/868154c56e82d1a8bdd1fc40331e32da7e8d4992/solc/CommandLineInterface.cpp#L981-L996

`StandardCompiler` does not, which makes it non-testable.

Reply:
Done.

Reply:
Please remember to add it to the issue though.

Reply:
Added already.